src/content/popup/tailwind.config.js

```diff
@@ -46,6 +50,14 @@ export default {
           'var(--base-font-size)',
           { lineHeight: 'calc(1.5 * var(--base-font-size))' },
         ],
+        kana: [
+          'calc(18 / 14 * var(--base-font-size))',
+          { lineHeight: 'calc(1.7 * var(--base-font-size))' },
+        ],
+        kanji: [
+          'calc(20 / 14 * var(--base-font-size))',
+          { lineHeight: 'calc(1.8 * var(--base-font-size))' },
+        ],
```

Comment:
How should these be named? Semantically, as they are currently, or according to the common tailwind sizes?

Reply:
I think it's best to name them based on Tailwind's size conventions, though I'm not entirely sure of the exact mappings.
If 14 is 'base,' then would `15 / 14` be lg, `16 / 14` xl, `17 / 14` 2xl ...?

Reply:
Yes, I agree. I wrote a few thoughts about this [over in the metadata PR](https://github.com/birchill/10ten-ja-reader/pull/2118#discussion_r1858306944).